pkgs/applications/misc/slstatus/default.nix

```diff
@@ -44,7 +44,7 @@ stdenv.mkDerivation rec {
 
   installFlags = [ "PREFIX=$(out)" ];
 
-  passthru.updateScript = gitUpdater { };
+  passthru.updateScript = nix-update-script { };
```

Comment:
Looking more closely into `nix-update`, this actually breaks here, because `nix-update` doesn't support arbitrary Git repos:

```
$ nix-shell maintainers/scripts/update.nix --argstr package slstatus
[...]
nix_update.errors.VersionError: Please specify the version. We can only get the latest version from
  codeberg/crates.io/gitea/github/gitlab/pypi/savannah/sourcehut/rubygems/npm projects right now
```

There may be others that would break, I didn't check, but it highlights that these are not interchangeable, and each has features the other doesn't.